### DISCUSSION TITLE
Flysky Hardware Support for ExpressLRS (EL18 and FRM303)

### DISCUSSION THREAD

**Discussion by richardclli** [ExpressLRS/ExpressLRS#2957] [General]
Opensource is freedom. Don't trying to dictate, control and backstabbing.

I have my rights to fork, modify and work on anything I feel interesting.

If you really to dictate and control, make it closed source!

https://github.com/richardclli/Flysky-ELRS

**Comment by JyeSmith** [comment]
As discussed, ELRS is no longer supporting STM hardware.

> I have my rights to fork, modify and work on anything I feel interesting.

No one stopping you.
